src/DNS/Adapter/Native.php

```
@@ -69,4 +69,19 @@ public function getName(): string
+     * Worker start callback
+     *
+     * @param callable $callback
+     * @return void
+     */
+    public function onWorkerStart(callable $callback): void
+    {
+        if (is_callable($callback)) {
+            call_user_func($callback, $this->server, 0); // Pass dummy worker ID
+        } else {
+            throw new Exception('Worker start callback must be callable.');
+        }
```

PHPMD (2.15.0)

The method onWorkerStart uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)